pkgs/development/python-modules/python-openems/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, cython
+, openems
+, csxcad
+, boost
+, python-csxcad
+, numpy
+, h5py
+}:
+
+buildPythonPackage rec {
+  pname = "python-openems";
+  version = "unstable-2020-02-15";
+
+  src = fetchFromGitHub {
+    owner = "thliebig";
+    repo = "openEMS";
+    rev = "ba793ac84e2f78f254d6d690bb5a4c626326bbfd";
+    sha256 = "1dca6b6ccy771irxzsj075zvpa3dlzv4mjb8xyg9d889dqlgyl45";
+  };
+
+  sourceRoot = "source/python";
+
+  nativeBuildInputs = [
+    cython
+    boost
+  ];
```

boost is usually linked against, and should go into buildInputs
```suggestion
  nativeBuildInputs = [
    cython
  ];
  
  buildInputs = [
    boost
  ];
```

---

although this is refering to the python package, not actually sure about this case

cc @FRidh